src/hybrid_build.jl

```diff
@@ -0,0 +1,297 @@
+import MathOptInterface
```

Comment:
Note for future refactor. Implement PSI's `add_variable` methods here. Add comment to that effect

Reply:
I only need this for adding the objective function. Any way of doing this using PSI?

```
JuMP.@objective(model, MOI.MAX_SENSE, container.objective_function.invariant_terms)
```

Reply:
no, you can open an issue in PSI to make a method to this so it can be reused.